integrations/atlassian/confluence/client.go

```diff
@@ -40,12 +40,12 @@ func New(cvars sdkservices.Vars) sdkservices.Integration {
 }
 
 // connStatus is an optional connection status check provided by
-// the integration to AutoKitteh. The possible results are "init
-// required" (the connection is not usable yet) and "using X".
+// the integration to AutoKitteh. The possible results are "Init
```

Comment:
This is from my PR #666 (😈). It shouldn't be a part of the review - is this a bad rebase of it? I'm not sure which other PRs got rolled into this one, so waiting for your ok to re-review this.

Reply:
looks like it's gone now